private-templates-service/client/src/components/SideBar/SideBar.tsx

```diff
@@ -181,7 +184,7 @@ const SideBar = (props: Props) => {
         {props.isAuthenticated && <NavMenu styles={navMenuLinksProps} groups={navMenuLinks} onLinkClick={onNavClick} />}
       </MainItems>
 
-      <SignOut onClick={props.authButtonMethod}>Sign {props.isAuthenticated ? "Out" : "In"}</SignOut>
+      <SignOut onClick={props.authButtonMethod} tabIndex={props.modalState ? -1 : 0}>Sign {props.isAuthenticated ? "Out" : "In"}</SignOut>
```

Comment:
What is this for?

Reply:
when a modal is open, users are unable to sign out by clicking on the sign out button, but if you use keyboard navigation, you can sign out by tabbing onto the sign out button.

this simply checks if there is a modal currently open and then if it is you can't tab onto the sign out button.

i did the same thing for all the buttons on template preview page